src/create-jobs.js

```diff
@@ -119,35 +125,92 @@ function checks (config) {
     return [
         new Check(
             config.code_checks,
-            [fileTest('phpunit.xml.dist'), fileTest('phpunit.xml')],
+            [fileTest('phpunit.xml.dist')],
```

Comment:
not sure if this is a good one.
This will probably create two checks in projects which do have both files.

Reply:
The scope of this was just to run `xmllint` against the various xml files.

I've just been cleaning up old PRs and issues and saw this was still outstanding. I'm unsure what change you want me to make here. If there should be a new script which alters how the .xml vs .xml.dist files are handled, I would suggest that should be a new issue with its own PR, and this blocked until that one is merged and rebased after? If that is sufficient, I can create a new PR to do that when I get some time (won't be soon).